cmd/lakectl/cmd/dbt.go

```diff
@@ -166,24 +118,24 @@ func copySchemaWithDbtTables(ctx context.Context, continueOnError, continueOnSch
 	case continueOnSchemaExists:
 		fmt.Printf("schema %s already exists, proceeding with existing schema\n", toSchema)
 	default:
-		DieFmt("schema %s exists, change schema or use %s flag", toSchema, continueOnSchemaFlag)
+		DieFmt("Schema %s exists, change schema or use %s flag", toSchema, continueOnSchemaFlag)
 	}
 
-	models := getDbtTables(dbtClient)
+	models := getDbtTables(projectRoot)
 	if err := copyModels(ctx, models, continueOnError, fromSchema, toSchema, branchName, dbfsLocation, client); err != nil {
 		DieErr(err)
 	}
 }
 
-func createViews(dbtClient dbtCommandClient, toSchema string) {
-	output, err := dbtClient.Run(toSchema, []string{"config.materialized:view"})
+func createViews(projectRoot, toSchema string) {
+	output, err := DbtRun(projectRoot, toSchema, lakectlIdentifier, []string{"config.materialized:view"}, commandExecutioner{})
```

Comment:
More nits:
```suggestion
	output, err := DBTRun(projectRoot, toSchema, lakectlIdentifier, []string{"config.materialized:view"}, commandExecutioner{})
```
(Can you go over the entire file and see we have no "Dbt", please?  (Can be in another PR).